src/main/scala/nl.knaw.dans.easy.deposit/StateManager.scala

```diff
@@ -84,8 +85,10 @@ case class StateManager(draftDeposit: DepositDir, submitBase: File, easyHome: UR
       }
   }
 
-  def setStateFailed(description: String): Try[Unit] = Try {
-    saveNewStateInDraftDeposit(StateInfo(State.submitted, description))
+  def setMailToDansDescription(state: State): Try[Unit] = Try {
+    // using saveNewStateInDraftDeposit could inadvertently change the state
+    draftProps.setProperty(stateDescriptionKey, mailToDansMessage)
+    draftProps.save()
```

Comment:
Why this `state: State` parameter? It's currently not used. From the usage I would expect a `draftsProps.setProperty(..., state)` somewhere here, which makes it equivalent to the original implementation.